phoenix-core/src/test/java/org/apache/phoenix/schema/MutationTest.java

```diff
@@ -92,18 +92,24 @@ public void testSizeConstraint() throws Exception {
                 fail();
             } catch (SQLException e) {
                 assertEquals(SQLExceptionCode.DATA_EXCEEDS_MAX_CAPACITY.getErrorCode(),e.getErrorCode());
+                assertFalse(e.getMessage().contains("abcd"));
+                assertTrue(e.getMessage().contains(String.valueOf(maxLength1)));
```

Comment:
nit: Same here. Could include the datatype too in the exception message and not just the length.